modules/createHashHistory.js

```diff
@@ -33,7 +37,7 @@ var DefaultQueryKey = '_k';
 function createHashHistory(options={}) {
   var { queryKey } = options;
 
-  if (queryKey && typeof queryKey !== 'string')
+  if (typeof queryKey !== 'string')
```

Comment:
I'd prefer to leave this line as it was so people can do `createHashHistory({ queryKey: true })` and get the default.